Resources/Locale/en-US/job/job-description.ftl

```diff
@@ -1,32 +1,32 @@
-job-description-chaplain = Preach the good word of your deity and religion, and conduct spiritual healing.
+job-description-chaplain = Preach the good word of your deity and religion, and conduct spiritual healing and miracles with your bible.
 job-description-chef = Keep the station fed with a variety of food items, butcher dead animals to ensure ammonia doesn't leak, and help keep the bar lively.
 job-description-chemist = Produce medicinal drugs for the doctors to use, research ethically dubious rare chemicals, and produce weapons of war when enemies of the station arrive.
 job-description-clown = Entertain the crew through elaborate slapstick routines or terrible jokes.
 job-description-cmo = Manage the resources and personnel of the medical department to keep the crew alive.
 job-description-paramedic = Rescue critically injured patients all over the station, sometimes outside too. Stablize them, give them a ride to medbay, and get back out there!
 job-description-detective = Investigate crime scenes using forensic tools, ensure that the guilty party is found, and have a couple smokes.
-job-description-doctor = Diagnose and heal crewmembers through medicinal chemicals, vaccinations, along with cloning the dead.
-job-description-engineer = Keep the station's main engine & solars active, optimize the power network, and make emergency repairs using your hardsuit.
+job-description-doctor = Diagnose and heal crewmembers through medicinal chemicals, advanced medicine, and defibrillators. Make sure the dead don't rot, and that cadavers are in the morgue.
+job-description-engineer = Keep the station's main engine & solars active, optimize the power network, and make emergency repairs using your hardsuit in spaced areas.
 job-description-ertengineer = Ensure that the station has power and clean air.
 job-description-ertjanitor = Ensure that the station is properly cleaned--for morale.
 job-description-ertleader = Lead the Emergency Response Team in dealing with threats to Nanotrasen assets.
 job-description-ertmedic = Ensure that the crew of the station is kept alive and healthy.
 job-description-ertsecurity = Ensure that any active threats to the station are put to rest.
-job-description-hop = Handle access reassignment fairly using your ID console, manage the service department, and keep Ian safe.
+job-description-hop = Handle access reassignment fairly using your ID console, manage demotions, change crews jobs on their request with other heads permissions, manage the service department, and keep Ian safe.
 job-description-hos = Manage your security force and keep them efficient, quell dissent, and keep the other heads of staff safe.
-job-description-intern = Learn the basics of administering medicine, cloning the dead, and creating chemicals.
+job-description-intern = Learn the basics of administering medicine, learning basic chemicals and using basic medical tools.
```

Comment:
Double mention of learning, the entire sentence structure already revolves around "Learn"